src/zoid/payment-fields/component.jsx

```diff
@@ -167,6 +167,18 @@ export function getPaymentFieldsComponent() : PaymentFieldsComponent {
                     value: getLocale
                 },
 
+                country: {
+                    type:          'object',
+                    queryParam:    'country.x',
+                    allowDelegate: true,
+                    queryValue({ value }) : string {
+                        // $FlowFixMe
+                        const { country } = value;
+                        return `${ country }`;
```

Comment:
```suggestion
                        return country;
```
Do you get a flow type error when just returning `country` without the string conversion? Hoping we can just return the variable and Flow is smart enough to consider the enum a string (https://github.com/paypal/paypal-sdk-constants/blob/d1314b517f3fcc65fd784d73f3a27a06b15087bb/src/locale.js#L667)

Reply:
@gregjopa  No error, it's fine returning the variable as is